From: SZALAY Attila <sasa@ubainba.hu>
Date: Sat, 8 Oct 2016 17:58:24 +0200
Subject: Do not use CPPFLAGS

Use AM_CPPFLAGS instead. The CPPFLAGS is reserved for users.
---
 modules/anypy/Makefile.am        | 2 +-
 modules/apr/Makefile.am          | 2 +-
 modules/finger/Makefile.am       | 2 +-
 modules/ftp/Makefile.am          | 2 +-
 modules/http/Makefile.am         | 2 +-
 modules/http/tests/Makefile.am   | 2 +-
 modules/plug/Makefile.am         | 2 +-
 modules/pop3/Makefile.am         | 2 +-
 modules/smtp/Makefile.am         | 2 +-
 modules/smtp/tests/Makefile.am   | 2 +-
 modules/telnet/Makefile.am       | 2 +-
 modules/telnet/tests/Makefile.am | 3 +--
 modules/whois/Makefile.am        | 2 +-
 tests/unit/Makefile.am           | 2 +-
 14 files changed, 14 insertions(+), 15 deletions(-)

diff --git a/modules/anypy/Makefile.am b/modules/anypy/Makefile.am
index 8271d13..d9146e9 100644
--- a/modules/anypy/Makefile.am
+++ b/modules/anypy/Makefile.am
@@ -1,7 +1,7 @@
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = AnyPy.py
 
diff --git a/modules/apr/Makefile.am b/modules/apr/Makefile.am
index 963f7a4..7805b28 100644
--- a/modules/apr/Makefile.am
+++ b/modules/apr/Makefile.am
@@ -3,7 +3,7 @@
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = APR.py
 
diff --git a/modules/finger/Makefile.am b/modules/finger/Makefile.am
index 29fd09b..6760ed2 100644
--- a/modules/finger/Makefile.am
+++ b/modules/finger/Makefile.am
@@ -1,7 +1,7 @@
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Finger.py
 
diff --git a/modules/ftp/Makefile.am b/modules/ftp/Makefile.am
index 66da8b0..04dfb88 100644
--- a/modules/ftp/Makefile.am
+++ b/modules/ftp/Makefile.am
@@ -3,7 +3,7 @@ SUBDIRS = tests
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Ftp.py
 
diff --git a/modules/http/Makefile.am b/modules/http/Makefile.am
index b794c83..56050c8 100644
--- a/modules/http/Makefile.am
+++ b/modules/http/Makefile.am
@@ -8,7 +8,7 @@ DIST_SUBDIRS += . messages tests
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 AM_CFLAGS = @CFLAGS@ -W
 
 pkgdata_DATA = Http.py
diff --git a/modules/http/tests/Makefile.am b/modules/http/tests/Makefile.am
index 67bc811..5ac4ae0 100644
--- a/modules/http/tests/Makefile.am
+++ b/modules/http/tests/Makefile.am
@@ -1,5 +1,5 @@
 LIBS=@MODULETESTS_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
+AM_CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
 
 check_PROGRAMS = http_parse_url http_canon_url
 
diff --git a/modules/plug/Makefile.am b/modules/plug/Makefile.am
index 6683db2..eca4c06 100644
--- a/modules/plug/Makefile.am
+++ b/modules/plug/Makefile.am
@@ -1,7 +1,7 @@
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Plug.py
 
diff --git a/modules/pop3/Makefile.am b/modules/pop3/Makefile.am
index 74e7b9c..b53bd48 100644
--- a/modules/pop3/Makefile.am
+++ b/modules/pop3/Makefile.am
@@ -3,7 +3,7 @@ SUBDIRS = messages
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Pop3.py
 
diff --git a/modules/smtp/Makefile.am b/modules/smtp/Makefile.am
index e240807..7928941 100644
--- a/modules/smtp/Makefile.am
+++ b/modules/smtp/Makefile.am
@@ -3,7 +3,7 @@ SUBDIRS = . tests
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Smtp.py
 
diff --git a/modules/smtp/tests/Makefile.am b/modules/smtp/tests/Makefile.am
index 5e796b2..94db606 100644
--- a/modules/smtp/tests/Makefile.am
+++ b/modules/smtp/tests/Makefile.am
@@ -1,6 +1,6 @@
 LIBS = @MODULETESTS_LIBS@ @DEPS_LIBS@
 LDADD = ../libsmtp.la -lboost_unit_test_framework
-CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
+AM_CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
 
 AM_DEFAULT_SOURCE_EXT = .cc
 
diff --git a/modules/telnet/Makefile.am b/modules/telnet/Makefile.am
index ab0e6b3..3826331 100644
--- a/modules/telnet/Makefile.am
+++ b/modules/telnet/Makefile.am
@@ -4,7 +4,7 @@ pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 
 LIBS     = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 
 pkgdata_DATA = Telnet.py
diff --git a/modules/telnet/tests/Makefile.am b/modules/telnet/tests/Makefile.am
index 92bda60..ceab0d9 100644
--- a/modules/telnet/tests/Makefile.am
+++ b/modules/telnet/tests/Makefile.am
@@ -1,6 +1,6 @@
 LIBS     = @MODULES_LIBS@
 LDADD = ../libtelnet.la -lboost_unit_test_framework
-CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
+AM_CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
 
 check_PROGRAMS = test_telnet
 check_SCRIPTS =
@@ -12,7 +12,6 @@ TESTS = $(check_PROGRAMS) $(check_SCRIPTS)
 TEST_EXTENSIONS = .py
 PY_LOG_COMPILER = $(PYTHON)
 
-CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
 test_telnet_LDADD = ../libtelnet.la @ZORP_LIBS@ -lboost_unit_test_framework
 
 EXTRA_DIST = $(check_SCRIPTS)
diff --git a/modules/whois/Makefile.am b/modules/whois/Makefile.am
index f8d0997..7704aa9 100644
--- a/modules/whois/Makefile.am
+++ b/modules/whois/Makefile.am
@@ -1,7 +1,7 @@
 pkgdatadir = @ZORP_PYTHON_DIR@
 pkglibdir = @libdir@/zorp
 LIBS = @MODULES_LIBS@
-CPPFLAGS = @MODULES_CPPFLAGS@
+AM_CPPFLAGS = @MODULES_CPPFLAGS@
 
 pkgdata_DATA = Whois.py
 
diff --git a/tests/unit/Makefile.am b/tests/unit/Makefile.am
index b960437..72db5ef 100644
--- a/tests/unit/Makefile.am
+++ b/tests/unit/Makefile.am
@@ -1,6 +1,6 @@
 LIBS = @ZORP_LIBS@ @MODULES_LIBS@ @DEPS_LIBS@
 LDADD = -lboost_unit_test_framework
-CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
+AM_CPPFLAGS = @MODULES_CPPFLAGS@ -DBOOST_TEST_DYN_LINK=1
 
 check_PROGRAMS = test_pystruct test_szig test_proxy
 
